rholang/src/main/scala/coop/rchain/rholang/interpreter/rholangADT.scala

```diff
@@ -12,22 +12,25 @@ case class Par(
     // selects: List[Select],
     evals: List[Eval],
     news: List[New],
-    exprs: List[Expr]
+    exprs: List[Expr],
     // matches: List[Match]
+    id: Option[GPrivate]
```

Comment:
I was under the assumption that we we're going to use

`case class GPrivate(p: String) extends Ground`

and thus these are just part of the exprs.

Reply:
We were, but it turns out, it's especially handy to be able to refer directly to an identifier, rather than having it blend with every other expression.

Reply:
Ok in that case can we remove the

`extends Ground `

then?

Reply:
@KentShikama Yes. Good catch.
@guardbotmk3 Why would a par need multiple par identifiers?